src/app/migration/src/src/app/modules/workspace/interfaces/collection.data.interface.ts

```diff
@@ -0,0 +1,11 @@
+// tslint:disable-next-line:class-name
+export class collectionDataInterface {
```

Comment:
this is class, why we need this

Reply:
Initially formmodules were used, now for reactive forms its not required. so i will remove now.